HIP-Basic/warp_shuffle/README.md

```diff
@@ -0,0 +1,49 @@
+# HIP-Basic Warp Shuffle Example
+
+## Description
+Kernel code is SIMT-executed by groups of threads within the same block, known as _warps_ (NVIDIA) or _wavefronts_ (AMD). Each block is divided into as many warps as the block's size allows, if the block size is less than the warp size then part of the warp just stays idle (as happens in this example). 
```

Comment:
SIMT or SIMD?, please define acronymn

Reply:
Rephrased this and the next paragraph a bit.

Reply:
See also the diagrams now added to this readme [here](https://github.com/amd/rocm-examples/tree/53bad130d49b27421e32478b1194a8723c0d2edc/HIP-Basic/warp_shuffle).